src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/Routes.scala

```diff
@@ -163,27 +163,24 @@ object Routes {
     )
     val appInfoRoutes = AppInfoRoutes(config.description, healthStatusGroup).routes
 
-    wrap(
-      pathPrefix(config.http.prefix / Segment) { resourceSegment =>
-        extractToken {
-          implicit optToken =>
-            project.apply {
-              implicit project =>
-                (extractCallerAcls & extractCaller) { (acl, c) =>
-                  resourceSegment match {
-                    case "resolvers" => new ResolverRoutes(resources, acl, c).routes
-                    case "views"     => new ViewRoutes(resources, acl, c).routes
-                    case "schemas"   => new SchemaRoutes(resources, acl, c).routes
-                    case "files"     => new FileRoutes(resources, acl, c).routes
-                    case "resources" => new ResourceRoutes(resources, acl, c).routes
-                    case "events"    => new EventRoutes(acl, c).routes
-                    case _           => reject()
-                  }
-                }
+    wrap(extractToken { implicit optToken =>
+      (extractCallerAcls & extractCaller) { (acl, c) =>
+        pathPrefix(config.http.prefix / Segment) {
```

Comment:
I would recommend using `concat` instead of `~` to join routes together; it makes stuff easier to read.